src/prommis/solid_handling/crusher.py

```diff
@@ -0,0 +1,216 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+r"""
+Crusher 
+=======
+
+Author: Lingyan Deng
+
+The Crusher module including power consumption for solid crushing. It is a function of particle size distribution, mass flow rate, and bond work index.
+
+Degrees of Freedom
+------------------
+
+A Crusher module has one degree of freedom, which is the "work" output. 
```

Comment:
This is not correct. The model has 2 degrees of freedom, which normally would be the product mean particle diameter and particle size width.

Degrees of freedom and the things we need to fix to get a square problem, work is the result we are calculating.